.Lib9c.Tests/Action/CombinationConsumableTest.cs

```diff
@@ -42,33 +42,64 @@ public void Execute()
             var agentState = new AgentState(_agentAddress);
             agentState.avatarAddresses[0] = _avatarAddress;
 
+            var worldSheet = new WorldSheet();
+            worldSheet.Set(_sheets[nameof(WorldSheet)]);
+            var questRewardSheet = new QuestRewardSheet();
+            questRewardSheet.Set(_sheets[nameof(QuestRewardSheet)]);
+            var questItemRewardSheet = new QuestItemRewardSheet();
+            questItemRewardSheet.Set(_sheets[nameof(QuestItemRewardSheet)]);
+            var equipmentItemRecipeSheet = new EquipmentItemRecipeSheet();
+            equipmentItemRecipeSheet.Set(_sheets[nameof(EquipmentItemRecipeSheet)]);
+            var equipmentItemSubRecipeSheet = new EquipmentItemSubRecipeSheet();
+            equipmentItemSubRecipeSheet.Set(_sheets[nameof(EquipmentItemSubRecipeSheet)]);
+            var questSheet = new QuestSheet();
+            questSheet.Set(_sheets[nameof(GeneralQuestSheet)]);
+            var characterSheet = new CharacterSheet();
+            characterSheet.Set(_sheets[nameof(CharacterSheet)]);
+            var consumableItemRecipeSheet = new ConsumableItemRecipeSheet();
+            consumableItemRecipeSheet.Set(_sheets[nameof(ConsumableItemRecipeSheet)]);
+            var materialItemSheet = new MaterialItemSheet();
+            materialItemSheet.Set(_sheets[nameof(MaterialItemSheet)]);
+            var worldUnlockSheet = new WorldUnlockSheet();
+            worldUnlockSheet.Set(_sheets[nameof(WorldUnlockSheet)]);
```

Comment:
It would be great to make test fixture for this because I guess it's used somewhere else too :)